nntrainer/tensor/manager.cpp

```diff
@@ -548,20 +530,14 @@ void Manager::deallocateInOuts() {
   shared_inout.deallocate();
 
   if (LAYER_V2) {
-    for (auto &li : inputs_v2) {
-      for (auto &in : li) {
-        in->deallocateVariable();
-      }
+    for (auto &in : inputs_v2) {
```

Comment:
Just checking, do we still have to have LAYER_V2 Conditional?

Reply:
this is cleaned up after 2 more PRs.